src/js/cart.js

```diff
@@ -45,6 +52,20 @@ function cartItemTemplate(item) {
   return newItem;
 }
 
+function deleteFromCart(event) {
+  const itemId = event.target.dataset.id;
+
+  if (!itemId) return;
+
+  const cartItems = getLocalStorage(cartLSKey) || [];
+
+  const updatedCart = cartItems.filter((item) => item.Id !== itemId);
+  localStorage.setItem(cartLSKey, JSON.stringify(updatedCart));
```

Comment:
@ashesmsmith I would suggest to use here setLocalStorage function from utils.mjs

Reply:
I made the change suggested. Do I need to delete this pull request and start a new one?

Reply:
no, you made a new commit to the current branch, it's enough.
great work!